package.json

```diff
@@ -37,6 +38,7 @@
     "eslint-plugin-html": "^6.0.0",
     "eslint-plugin-svelte3": "^2.7.3",
     "nightwatch": "^1.3.4",
+    "nodemon": "^2.0.2",
```

Comment:
Please don't add dependencies that aren't strictly necessary for this PR.

Speaking of dependencies, you might need to bump chromedriver to v80 for the CI to succeed at this stage (looks like the GH Actions servers have a new version of Chrome installed now)